packages/block-editor/src/store/test/selectors.js

```diff
@@ -4205,382 +4201,6 @@ describe( 'selectors', () => {
 		} );
 	} );
 
-	describe( '__experimentalGetAllowedPatterns', () => {
```

Comment:
These tests are moved to a separate file (above), because they now depend on a registered store.